### PR TITLE
Add documentation for match function

### PR DESCRIPTION
**Pull request by wbiller**
@brian-brazil Added the documentation for the match function I added to alertmanager.

Signed-off-by: Waldemar Biller <wbiller@gmail.com>

### CODE REVIEW

File: content/docs/alerting/notifications.md [hunk 1]
```diff
@@ -86,3 +86,4 @@ templating.
 | toLower | string | [strings.ToLower](http://golang.org/pkg/strings/#ToLower), converts all characters to lower case. |
 | safeHtml | text string | [html/template.HTML](https://golang.org/pkg/html/template/#HTML), Marks string as HTML not requiring auto-escaping. |
 |reReplaceAll | pattern, replacement, text | [Regexp.ReplaceAllString](http://golang.org/pkg/regexp/#Regexp.ReplaceAllString) Regexp substitution, unanchored. |
+| match | pattern, string | [Regexp.MatchString](https://golang.org/pkg/regexp/#MatchString). Match a string using Regexp. |
```
Comment (brian-brazil) [inline content/docs/alerting/notifications.md]: Can we keep the ordering the same as for https://prometheus.io/docs/prometheus/latest/configuration/template_reference/#strings?

Comment (wbiller) [reply content/docs/alerting/notifications.md]: yes.

Comment (wbiller) [reply content/docs/alerting/notifications.md]: I also moved toUpper, toLower to the same place as they're at the prometheus docs.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**